optimum/intel/ipex/inference.py

```diff
@@ -40,6 +77,55 @@ def __getattr__(self, item):
             return self.item
 
 
+class _ModelGenerationWrapper(GenerationMixin):
+    __slots__ = ("_optimized", "_default")
+
+    def __init__(self, optimized, default):
+        self._optimized = optimized
+        self._default = default
+
+    def __call__(self, *args, **kwargs):

```

Comment:
We should replace it with https://github.com/huggingface/optimum-intel/pull/283 before merging no ?